pipelines/pingcap/tidb/latest/pod-ghpr_build.yaml

```diff
@@ -27,31 +27,25 @@ spec:
         - name: bazel-out-lower
           subPath: tidb/go1.19.2
           mountPath: /bazel-out-lower
-        - mountPath: /bazel-out-overlay
-          name: bazel-out-overlay
+        - name: bazel-out-overlay
+          mountPath: /bazel-out-overlay
+        - name: bazel-repository-cache
+          mountPath: /share/.cache/bazel-repository-cache
         - name: gocache
           mountPath: /share/.cache/go-build
         - name: gopathcache
           mountPath: /share/.go
+        - name: bazel-rc
+          mountPath: /data/
+          readOnly: true
         - name: containerinfo
           mountPath: /etc/containerinfo
       lifecycle:
         postStart:
           exec:
             command:
               - /bin/sh
-              - -c
-              - mkdir -p /bazel-out-overlay/upper /bazel-out-overlay/work; 
-                sudo mount -t overlay overlay /home/jenkins/.tidb/tmp -o lowerdir=/bazel-out-lower,upperdir=/bazel-out-overlay/upper,workdir=/bazel-out-overlay/work;
-
-                cpu_limit=$(cat /etc/containerinfo/cpu_limit);
-                mem_limit=$(cat /etc/containerinfo/mem_limit);
-                mem_limit=$(((mem_limit / 1048576) * 9 / 10 ));
-                :> /data/bazel; 
-                echo "build:ci --remote_cache=http://brc-bazel-remote.apps.svc --remote_timeout=10s" >> /data/bazel;
-                echo "build:ci --local_ram_resources=${mem_limit} --local_cpu_resources=${cpu_limit} -j ${cpu_limit}" >> /data/bazel;  
-                echo "test:ci --remote_cache=http://brc-bazel-remote.apps.svc --remote_timeout=10s" >> /data/bazel;
-                echo "test:ci --local_ram_resources=${mem_limit} --local_cpu_resources=${cpu_limit} -j ${cpu_limit}"  >> /data/bazel;
+              - /data/bazel-prepare-in-container.sh
```

Comment:
where is ```bazel-prepare-in-container.sh``` ?

Reply:
from the mounted k8s secret, source at: https://github.com/PingCAP-QE/ee-ops/blob/main/apps/prod/jenkins/post/_base/files/bazel-prepare-in-container.sh

Reply:
same with current running, but only extracted and optimized the shells from k8s pod YAML.